content/shared/influxdb3-query-guides/snapshots/snowflake.md

```diff
@@ -0,0 +1,140 @@
+# Integrate with Snowflake using Apache Iceberg
+
+
+Export time-series data snapshots from InfluxDB into Apache Iceberg format.
+Integrate data with Snowflake and other Iceberg-compatible tools without the need for complex ETL processes.
+
+### Key Benefits
+
+- **Efficient data access**: Query InfluxDB data directly from Snowflake.
+- **Cost-effective storage**: Optimize data retention and minimize storage costs.
+- **Supports AI and ML workloads**: Enhance machine learning applications by making time-series data accessible in Snowflake.
+
```

Comment:
We should remove the "Step <num>" from the headings and add a numbered-list TOC for the steps here.